### PR TITLE
75 update user escalation policy

### PR DESCRIPTION
**Pull request by kalebdavis**
Fixes https://github.com/PagerNation/PagerNation/issues/75

### CODE REVIEW

File: server/services/user.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import Joi from 'joi';
+import _ from 'lodash';
```
Comment (jarrydlee) [inline server/services/user.js]: You don't need this import

File: server/services/user.js [hunk 2]
```diff
@@ -29,7 +30,7 @@ function updateUser(userId, userObject) {
     email: Joi.string().email(),
     auth: Joi.string(),
     groups: Joi.array().items(Joi.string().hex().length(24)),
-    devices: Joi.array(), // TODO this needs to be updated once the device model is done
+    devices: Joi.array().items(DEVICE_SCHEMA).options({ allowUnknown: true }),
```
Comment (jarrydlee) [inline server/services/user.js]: I don't think we should be allowing unknown here, we should know exactly what we are expecting.

File: server/services/user.js [hunk 3]
```diff
@@ -116,5 +117,5 @@ export default {
   // User Group Modifications
   addGroupByUserId,
   getGroupsForUser,
-  removeGroup
+  removeGroup,
```
Comment (jarrydlee) [inline server/services/user.js]: Extra comma

File: server/services/user.js (hunk 2 above)
Comment (kalebdavis) [inline server/services/user.js]: Would you be satisfied with `stripUnknown`? We're passing in a mongo object here, which has a bunch of keys under the hood.

**Comment by jarrydlee** [comment]
Apart from that, all good

**Pull request merged**